frontend/src/components/fundingAgreements/ViewFundingAgreement.vue

```diff
@@ -219,6 +239,28 @@ export default {
       }
     },
 
+    async loadFundingAgreementLicences() {
+      this.isLicencesLoading = true;
```

Comment:
I don't think we need a separate flag for the licences. In fact, I find it looks odd because the bottom of the screen loads before the top. I think we can just use one flag.